versions/1.0.0.md

```diff
@@ -257,7 +257,7 @@ Field Name | Type | Description
 <a name="workflowId"></a>workflowId | `string` | **REQUIRED**. Unique string to represent the workflow. The id MUST be unique amongst all workflows describe in the Workflows document. The `workflowId` value is **case-sensitive**. Tools and libraries MAY use the `workflowId` to uniquely identify a workflow, therefore, it is RECOMMENDED to follow common programming naming conventions.
 <a name="workflowSummary"></a>summary | `string` | A summary of the purpose or objective of the workflow.
 <a name="workflowDescription"></a>description | `string` | A description of the workflow. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
-<a name="workflowInputs"></a>inputs | `JSON Schema` or `string` | A JSON Schema object representing the input parameters used by this workflow.  If a `string` is provided, it must correspond to the key of the [[Component Object]](#component-object)'s `inputs` field.
+<a name="workflowInputs"></a>inputs | `JSON Schema` | A JSON Schema 2020-12 object representing the input parameters used by this workflow. Some of the input parameters may be [References](#reference-object) to the [[Component Object]](#component-object)'s `inputs` field.

```

Comment:
If this proposal is trying for the same feel as OAS 3.1, then you don't need to talk about Reference Objects here.  Reference Objects are used  _outside_ of Schema Objects, where `$ref` is not otherwise defined.  The OAS 3.1 Schema Object is just a regular 2020-12 schema, and `$ref` is already defined with behavior specific to schemas (in terms of how adjacent keywords are handled).

Reply:
Hmm ok, I can take out that sentence. I'll leave the example in though.